xonsh/execer.py

```diff
@@ -68,7 +68,8 @@ def parse(self, input, ctx, mode='exec'):
         # tokens for all of the Python rules. The lazy way implemented here
         # is to parse a line a second time with a $() wrapper if it fails
         # the first time. This is a context-free phase.
-        tree, input = self._parse_ctx_free(input, mode=mode)
+        tree, input = self._parse_ctx_free(input, mode=mode,
+                                           wrap_subprocs=wrap_subprocs)
```

Comment:
Oooooooh.  Okay.  I was confused by your comments about this initially, but this cleared it up.  I can make that change.